x-pack/qa/kerberos-tests/build.gradle

```diff
@@ -57,12 +63,5 @@ integTest.runner {
             "-Djava.security.krb5.conf=${project(':test:fixtures:krb5kdc-fixture').ext.krb5Conf("peppa")}",
             "-Dsun.security.krb5.debug=true"
     ])
+    classpath += copyKeytabToGeneratedResources.outputs.files
```

Comment:
Is this what effectively depends on the task to run at runtime?

Reply:
Yes. Since `classpath` is a task input and `outputs.files` is a `Buildable` this establishes an implicit dependency.

Reply:
I'm wondering if `files(copyKeytabToGeneratedResources)` would be more concise and clear. They are both effectively the same. Preference?